templates/dex-k8s-authenticator.yaml

```diff
@@ -23,8 +23,10 @@ spec:
       kubeaddons.mesosphere.io/name: dex
   chartReference:
     chart: dex-k8s-authenticator
-    repo: https://mesosphere.github.io/charts/staging
-    version: 1.1.6
+    # TODO(mh): Point to main mesosphere repo once charts PR gets merged
+    # repo: https://mesosphere.github.io/charts/staging
+    repo: https://timaa2k.github.io/charts
+    version: 1.1.7
```

Comment:
To allow dex-k8s-authenticator to use `HTTP_PROXY` settings, we'll be releasing an updated image, which will need to be tagged in this file.